test/detect-stdfs.cc

```diff
@@ -0,0 +1,18 @@
+// Formatting library for C++ - tests of formatters for standard library types
+//
+// Copyright (c) 2012 - present, Victor Zverovich
+// All rights reserved.
+//
+// For the license information refer to format.h.
+
+#include <exception>
```

Comment:
This include looks unused.

Reply:
The macros ``_GLIBCXX_RELEASE`` and ``_LIBCPP_VERSION`` is defined in standard library header files.

``<exception>`` is the smallest of the standard library header files.

Reply:
I see. Maybe include `<initializer_list>` which is slightly cheaper according to https://artificial-mind.net/projects/compile-health/ and add a comment that this is for `_GLIBCXX_RELEASE` & `_LIBCPP_VERSION`?

Reply:
``<initializer_list>`` is C++11.
Specifying the version of the standard will complicate the call to ``try_compile``.

Comment added.